docs/source/index.rst

```diff
@@ -57,6 +67,17 @@ On Ubuntu 14.04 (Trusty):
   :linenos:
   :lines: 20-
 
+System Requirements
+===================
+
+Although Cartographer may run on other systems, it is confirmed to be working
```

Comment:
Perhaps we should make note of that as a known limitation. 

Proposed wording:
https://github.com/catskul/cartographer/commit/9561cc6ad8f1f883ce6fbbb8c573d12609ac3957

If we wanted to be thorough, we could add in a check for 32bit in CMakeLists.txt with an error message to the same effect.